kemp-loadmaster-ha-pair/azuredeploy.json

```diff
@@ -0,0 +1,388 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmNamePrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Prefix to use for LoadMaster names"
+      }
+    },
+    "balPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for bal account"
+      }
+    },
+    "loadBalancerName": {
+      "type": "string",
+      "metadata": {
+        "description": "This is the name of the Layer 4 Azure load balancer used in HA Pair"
+      }
+     }
+    },
+  "variables": {
+    "availabilitySetName": "AvSet",
+    "adminUsername": "bal",
+    "adminaccountname": "bal",
+    "storageAccountType": "Standard_LRS",
+    "virtualNetworkName": "VLM_Network",
+    "subnetName": "Subnet1",
+    "vmSize": "Standard_A1",
+    "networkinterfacename": "[concat(parameters('vmNamePrefix'),'-NIC')]",
+    "networksecurityname": "[concat(parameters('vmNamePrefix'),'-NSG')]",
+    "publicIPAddressName": "[concat(parameters('vmNamePrefix'),'-PIP')]",
+    "dnsNameforLBIP": "[concat('vlm',uniqueString(resourceGroup().id))]",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables ('subnetName'))]",
+    "numberOfInstances": 2,
+    "lbID": "[resourceId('Microsoft.Network/loadBalancers',parameters('loadBalancerName'))]",
+    "imageSKU": "basic-byol",
+    "publicIPAddressType": "Dynamic",
+    "publicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
+    "imageOffer": "vlm-azure",
+    "imagePublisher": "kemptech",
+    "imageVersion": "latest",
+    "storageAccountName": "[concat('vhds', uniqueString(resourceGroup().id))]",
+    "frontEndIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/loadBalancerFrontEnd')]",
+    "imageReference": {
+  	"publisher": "[variables('imagePublisher')]",
+  	"offer": "[variables('imageOffer')]",
+  	"sku": "[variables('imageSku')]",
+  	"version": "[variables('imageVersion')]"
+     },
+    "imagePlan": {
+ 	"name": "[variables('imageSku')]",
+  	"product": "[variables('imageOffer')]",
+  	"publisher": "[variables('imagePublisher')]"
+     }
+   },
+  "resources": [
```

Comment:
run this through a json code formatter (e.g. VS or VS Code) to clean up white space and indention